### PR TITLE
Nikolai.berestevich signal provider highlight 28666

### PR DESCRIPTION
**Pull request by nikolaiberestevich-deriv**
Changes:

-  styled component doesn't rerender with new props during the first loading of the page. So i added new variable to fix it

## Type of change

-   [x ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by arome-binary — APPROVED** [review]
LGTM

File: src/pages/dmt5-trading-signals/index.js [hunk 1]
```diff
@@ -102,6 +102,10 @@ const Separator = styled.div`
 
 const DMT5TradingSignals = () => {
     const [active_tab, setActiveTab] = useTabState(['signal-subscriber', 'signal-provider'])
+    const [not_first_loading, setNotFirstLoading] = useState(false) //needs to fix bug with hightlight of the 1st loading
```
Comment (sean-binary) [inline src/pages/dmt5-trading-signals/index.js:105]:
```suggestion
    const [is_mounted, setMounted] = useState(false) // needed to fix tab highlighting not being rerendered during first load
```

Comment (sean-binary) [reply src/pages/dmt5-trading-signals/index.js:105]: A suggestion for renaming the variable to indicate that it is called once the whole component has mounted.

**Review by sean-binary — DISMISSED** [review]
Rest LGTM

**Comment by nikolaiberestevich-deriv** [comment]
change names of variables (as Sean suggested)

**Review by sean-binary — APPROVED** [review]

**Pull request merged**